crates/ty_python_semantic/src/types/function.rs

```diff
@@ -707,9 +692,6 @@ pub(super) fn walk_function_type<'db, V: super::visitor::TypeVisitor<'db> + ?Siz
     visitor: &V,
 ) {
     walk_function_literal(db, function.literal(db), visitor);
-    for mapping in function.type_mappings(db) {
-        walk_type_mapping(db, mapping, visitor);
-    }
 }
```

Comment:
do we not need to walk the new fields here (`updated_signature` and `updated_last_definition_signature`)? Would it be worth adding a comment about why that's not necessary?